src/components/map-components/expansion-panel.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import MuiExpansionPanel from '@material-ui/core/ExpansionPanel';
+import MuiExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import MuiExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import { withStyles } from '@material-ui/core/styles';
+
+const ExpansionPanel = withStyles({
+  root: {
+    border: '1px solid rgba(0, 0, 0, .125)',
```

Comment:
im a bit worried about having a mix of JSS and scss styling but its a larger issue we'll have to figure out holistically